### DISCUSSION TITLE
other qrcode formats

### DISCUSSION THREAD

**Discussion by a1git** [chillerlan/php-qrcode#87] [Q&A]
Hi, 

checking if you have any ideas on how to produce custom qr designs  like in *link redacted*

**Comment by codemasher** [comment]
sorry, i don't click random links - if you want to show something, attach a screenshot please.

**Comment by a1git** [comment]
![qrcodes](https://user-images.githubusercontent.com/4275664/135449310-1c200cb9-5912-4836-ae0c-758d8f646322.png)

**Reply by a1git** [reply]
like these

**Reply by codemasher** [reply]
Several of these are doable, like the flags top right or the one bottom right or even the pixel art ones. Some look like the logo is just slapped on top of the qrcode, which is certainly doable too. I do like the tiny dot ones, i think this could be done with SVG or maybe Imagick (the GD library only allows for ellipses).
If you mean by "ideas" that i would implement any of these: certainly not - that's not really the focus of this library. However, it's flexible enough that most of these *can* be done.

**Comment by codemasher** [comment]
See:
- https://github.com/chillerlan/php-qrcode/blob/main/examples/QRImageWithLogo.php
- https://github.com/chillerlan/php-qrcode/issues/23
